components/automate-ui/src/app/services/projects-filter/projects-filter.effects.ts

```diff
@@ -21,15 +24,18 @@ import {
 export class ProjectsFilterEffects {
   constructor(
     private actions$: Actions,
+    private store: Store<NgrxStateAtom>,
     private projectsFilter: ProjectsFilterService,
     private requests: ProjectsFilterRequests
   ) { }
 
   private POLLING_INTERVAL_IN_SECONDS = 120; // 2 minutes
 
   @Effect()
-  latestOptions$ = observableInterval(1000 * this.POLLING_INTERVAL_IN_SECONDS)
-    .pipe(mergeMap(this.loadOptionsAction$()));
+  latestOptions$ = observableInterval(1000 * this.POLLING_INTERVAL_IN_SECONDS).pipe(
+    withLatestFrom(this.store.select(atLeastV2p1)),
+    filter(([_, projectsEnabled]) => projectsEnabled),
```

Comment:
Note that I deliberately named the selector itself with an explicit version (`atLeastV2p1`), because it does convey some useful info to the reader, and named its value as `projectsEnabled`, showing what it is used for without having to add any comments.